core/sqf/monitor/linux/cmsh.cxx

```diff
@@ -127,11 +127,14 @@ int CCmsh::GetClusterState( PhysicalNodeNameMap_t &physicalNodeMap )
             it = physicalNodeMap.find( nodeName.c_str() );
             if (it != physicalNodeMap.end())
             {
-                // TEST_POINT: to force state down on node name 
+               // TEST_POINT and Exclude List : to force state down on node name 
                 const char *downNodeName = getenv( TP001_NODE_DOWN );
-                if ( downNodeName != NULL && 
-                    !strcmp( downNodeName, nodeName.c_str() ) )
-                {
+                const char *downNodeList = getenv( TRAF_EXCLUDE_LIST );
+                if (( downNodeList != NULL && 
+                    (strstr(downNodeList,nodeName.c_str()))) ||
```

Comment:
I suppose TRAF_EXCLUDE_LIST is a debugging tool. It occurs to me though that strstr() could return false positives, for example if there were two nodes  n001.x.y and another nn001.x.y. If nn001.x.y were in the list, strstr of n001.x.y would return true. But maybe we don't care about this for debugging purposes?